app/models/attribute.rb

```diff
@@ -64,7 +64,9 @@ def find_localization(locale, id, key)
     #
     # `dist/` serialization
 
-    def as_json(attributes, version:, locale: "en")
+    def as_json(attributes, version:, locale: "en", raw_attributes:)
```

Comment:
`raw_attributes` isn't super intuitive to me in isolation. Is there a better representation within the context of the `Attribute` model?